
=== Running experiment: MultipleNature_Nature_Freq4_Den1.0_Type3_ObsErr1 with NTemp=2 ===

iteration 1 of 117
Reading observations from file  /home/jorge.gacitua/experimentos/L96_multiple_experiments/data/Nature/Nature_JRMultipleNature_Nature_Freq4_Den1.0_Type3_ObsErr1.npz
Traceback (most recent call last):
  File "./Orig_obs_error_sensitivity.py", line 75, in <module>
    result = alm.assimilation_letkf_run(conf)
  File "/home/jorge.gacitua/experimentos/L96_multiple_experiments/codes_experiments/assimilation_letkf_module.py", line 39, in assimilation_letkf_run
    InputData=np.load(GeneralConf['ObsFile'],allow_pickle=True)
  File "/opt/intel/oneapi/intelpython/latest/lib/python3.7/site-packages/numpy/lib/npyio.py", line 417, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: '/home/jorge.gacitua/experimentos/L96_multiple_experiments/data/Nature/Nature_JRMultipleNature_Nature_Freq4_Den1.0_Type3_ObsErr1.npz'
